packages/botonic-cli/README.md

```diff
@@ -21,7 +21,7 @@ $ npm install -g @botonic/cli
 $ botonic COMMAND
 running command...
 $ botonic (-v|--version|version)
-@botonic/cli/0.18.5 darwin-x64 node-v12.21.0
+@botonic/cli/0.18.6-alpha.0 darwin-x64 node-v14.16.1
```

Comment:
Why the node version is changing here?

Reply:
nice catch! this is something that is generated automatically by oclif. In practical terms it means I have ran the publish command with node 14 in my local machine. I would downgrade to node 12 when publishing the final package.